index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+
+
+    <header>
+        <div class="topnav">
+            <nav>
+                <li><a href=""><i class="fa-brands fa-instagram-square"></i></a></li>
+                <li><a href=""> <i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+            </nav>
+        </div>
+        <div class="mainnav">
+            <h1 class="main-h1"> The chance</h1>
+            <nav>
+                <li><a href="/about.html"> About</a> </li>
+                <li> Contact</li>
+                <li> Sponsor</li>
+                <li>News</li>
+                <button class="main-button"> Buy Ticket now</button>
+            </nav>
+        </div>
```

Comment:
- [ ] Kindly be noticed that the logo that should be shown on the lift side of the navbar is not shown please fix that.
- [ ] Also please add a link to the home page in the navbar.
- [ ] Also make the logo to be a link to the home page.